modules/table/src/test/java/org/apache/ignite/internal/table/KeyValueViewOperationsSimpleSchemaTest.java

```diff
@@ -84,7 +86,7 @@ public void put() {
         assertEquals(22L, tbl.get(null, 1L));
 
         // Remove KV pair.
-        tbl.put(null, 1L, null);
+        tbl.remove(null, 1L);
```

Comment:
By the way, what is the behavior of the `put` method after this commit in the case of the `null` value?

Reply:
`put` accepts `null` values.
this case covered in getOrDefault() and getNullable() tests.

Reply:
Hmm... What is a semantic of `put(tx, key, null)` invocation in the case of `KeyValueView` or `RecordView`?